frontend/src/components/MainPannel/MobileMainPannel.tsx

```diff
@@ -12,7 +12,9 @@ import MobileBulletScriptRow from 'components/MainPannel/MobileBulletScriptRow';
 
 const useStyles = makeStyles(() => ({
     fullHeightMinusBar: {
-        height: 'calc(calc(var(--vh, 1vh) * 100) - 74px)',
+        // 50px : the (menu) top bar
+        // 74px : the (menu) bottom bar
+        height: 'calc(calc(var(--vh, 1vh) * 100) - 50px - 74px)',
```

Comment:
Maybe add a commentary for the future, I can't really understand what that CSS does